common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/protocol/BlockTransferMessage.java

```diff
@@ -35,6 +35,9 @@
  *     shuffle service. It returns a StreamHandle.
  *   - UploadBlock is only handled by the NettyBlockTransferService.
  *   - RegisterExecutor is only handled by the external shuffle service.
+ *   - StreamHandle is only handled by the OneForOneBlockFetcher.
+ *   - RegisterDriver is only handled by the MesosExternalShuffleService.
+ *   - HearBeat is only handled by the MesosExternalShuffleService.
```

Comment:
Is it right to modify them like below ?

StreamHandle is only handled by  the external shuffle service
RegisterDriver & OpenBlock is only handled by the external shuffle service while the cluster in the mesos deploy mode.